lib/apiservers/portlayer/restapi/handlers/containers_handlers.go

```diff
@@ -436,28 +428,41 @@ func convertContainerToContainerInfo(container *exec.ContainerInfo) *models.Cont
 		}
 	}
 
-	path := container.ExecConfig.Sessions[ccid].Cmd.Path
-	info.ProcessConfig.ExecPath = &path
+	// in heavily loaded environments we were seeing a panic due to a missing
+	// session id in execConfig -- this has only manifested itself in short lived containers
+	// that were initilized via run
```

Comment:
I think I didn't understand this comment. Do we know why session id is missing? It sounds like a corruption or race somewhere and if that's the case this is just a workaround